swagger.json

```diff
@@ -959,6 +959,182 @@
                 }
             }
         },
+        "/fleet/unassigned_driving_segments": {
+            "get": {
+                "tags": [
+                    "Fleet", "Default"
+                ],
+                "summary": "/fleet/unassigned_driving_segments",
+                "description": "Get the unassigned driving segments for a specified range.",
+                "operationId": "get_fleet_unassigned_driving_segments",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/accessTokenParam"
+                    },
+                    {
+                        "name": "start_ms",
+                        "in": "path",
+                        "required": true,
+                        "description": "Beginning of the time range, specified in milliseconds UNIX time.",
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                    {
+                        "name": "end_ms",
+                        "in": "path",
+                        "required": true,
+                        "description": "End of the time range, specified in milliseconds UNIX time.",
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Unassigned driving segments for duration.",
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "unassigned_driving_segments": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/definitions/UnassignedDrivingSegment"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/fleet/unassigned_driving_segments/annotate": {
```

Comment:
This should be a `PATCH /fleet/unassigned_driving_segments/:id` rather than `PATCH /fleet/unassigned_driving_segments/annotate` (verb should be the HTTP method, not a path parameter).

Reply:
Similar as the other comment, do you mean
`/fleet/unassigned_driving_segments/:id/annotate`

Reply:
No, I think the annotation is as simple as PATCH updating the `annotation` field of the unassigned driving segment. Is the annotation a simple string attribute of the driving segment, or a timeseries value?